tests/CMakeLists.txt

```diff
@@ -341,6 +341,52 @@ if(BUILD_CONVERT_V5)
 		target_link_libraries(pmemlog_15 PRIVATE -Wl,--version-script=${CMAKE_SOURCE_DIR}/nvml-1.5/src/libpmemlog/libpmemlog.map)
 	endif()
 
+	set(PMEMSPOIL_SOURCES
+                ${CMAKE_SOURCE_DIR}/nvml-1.5/src/test/tools/pmemspoil/spoil.c
+                ${CMAKE_SOURCE_DIR}/nvml-1.5/src/tools/pmempool/common.c
+                ${CMAKE_SOURCE_DIR}/nvml-1.5/src/libpmemblk/btt.c
+                ${CMAKE_SOURCE_DIR}/nvml-1.5/src/tools/pmempool/output.c
+                ${COMMON_SOURCES}
+		)
+
+	if(WIN32)
+	add_executable(pmemspoil ${PMEMSPOIL_SOURCES} ${CMAKE_SOURCE_DIR}/nvml-1.5/src/windows/getopt/getopt.c)
+
+	target_include_directories(pmemspoil PRIVATE ${CMAKE_SOURCE_DIR}/nvml-1.5/src/windows/getopt)
+	target_include_directories(pmemspoil PRIVATE ${CMAKE_SOURCE_DIR}/nvml-1.5/src/windows/include)
+	else()
+	add_executable(pmemspoil ${PMEMSPOIL_SOURCES})
```

Comment:
You don't have to duplicate add_executable in both cases. Append getopt.c to PMEMSPOIL_SOURCES in WIN32 path and use single add_executable after.